extensions/roc-plugin-dredd/src/roc/index.js

```diff
@@ -1,7 +1,32 @@
+import { lazyFunctionRequire } from 'roc';
+
 import config from '../config/roc.config.js';
 import meta from '../config/roc.config.meta.js';
 
+const lazyRequire = lazyFunctionRequire(require);
+
 export default {
     config,
-    meta
+    meta,
+    actions: [
+        {
+            hook: 'server-started',
+            description: 'Run dredd tests',
+            action: lazyRequire('../actions/dredd'),
```

Comment:
This action will be invoked each time the server restarts, that will be triggered by default via modifications to the code, the `roc.config` or the `/config` directory. 

Does the action itself manage this currently? Should the current Dredd run be canceled in that case and restart it or should all of those modifications be ignored?